sklearn/_build_utils/cythonize.py

```diff
@@ -28,6 +28,8 @@
 
 Note: this script does not check any of the dependent C libraries; it only
 operates on the Cython .pyx files.
+
+.. versionadded:: 0.18
```

Comment:
I don't think this is ever rendered / interpreted. It doesn't hurt either, though.